utils/das_data_sorter.py

```diff
@@ -20,39 +23,48 @@
 reed_vel = data["reed_velocity"]
 reed_dis = data["reed_distance"]
 
-# sorts time into seconds
-seconds = []
+time = []
+if args.unit == "seconds" or args.unit == "s":
+    time = milliseconds/1000
+elif args.unit == "minutes" or args.unit =="m":
+    time = milliseconds/1000/60
+
+# sorts time into specific time unit
+new_time = []
 previous_time = 0
 for i in range(len(time)):
     if time[i] > previous_time:
         # sets new previous_time, pushes current millisecond_indices into seconds array, creates new millisecond_indices
-        # millisecond_indices is an array of indexes for milliseconds within the same second
+        # millisecond_indices is an array of indexes for milliseconds within the same specified unit
         if previous_time != 0:
             # ignores first iteration
-            seconds.append(millisecond_indices)
+            new_time.append(millisecond_indices)
+
 
         previous_time = ceil(time[i])
         millisecond_indices = []
 
     # pushes index into current millisecond_indices
     millisecond_indices.append(i)
-seconds.append(millisecond_indices)
+
+new_time.append(millisecond_indices)
 
 # averages data within the seconds, according to seconds array 
 def avg_Data(array):
     avg_array = []
-    for second in seconds:
+
+    for t in new_time:
         total = 0
-        for i in second:
+        for i in t:
```

Comment:
I would also rename `i` to something else. `actual_millisecond`? Your decision! Hope you get the idea.